src/components/CopyTextToClipboard.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useCallback} from 'react';
+import {StyleProp, TextStyle} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import Clipboard from '@libs/Clipboard';
+import * as Expensicons from './Icon/Expensicons';
+import PressableWithDelayToggle from './Pressable/PressableWithDelayToggle';
+
+type CopyTextToClipboardProps = {
+    /** The text to display and copy to the clipboard */
+    text: string;
+
+    /** Styles to apply to the text */
+    textStyles?: StyleProp<TextStyle>;
+
+    urlToCopy?: string;
+};
+
+function CopyTextToClipboard(props: CopyTextToClipboardProps) {
+    const {translate} = useLocalize();
+
+    const copyToClipboard = useCallback(() => {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
I understand why you are disabling the rule, but you could add a comment explaining why it's necessary.

Reply:
Updated!